eacl/types.go

```diff
@@ -92,3 +94,12 @@ func (u *ValidationUnit) WithEACLTable(table *Table) *ValidationUnit {
 
 	return u
 }
+
+// WithAccount configures ValidationUnit to use as sender's [user.ID] bytes.
+func (u *ValidationUnit) WithAccount(v []byte) *ValidationUnit {
```

Comment:
shouldnt it be `user.ID` too? why it differs from `Target`?

Reply:
yes, it should. Updated